src/components/Participant/Participant.js

```diff
@@ -30,15 +34,54 @@ function unsetVideo(videoRef) {
 }
 
 function toggleFocus(id, focus) {
-  return focus === 'user' ? participantsActions.unsetFocus() : participantsActions.setFocus(id);
+  return isFocused(focus) ? participantsActions.unsetFocus() : participantsActions.setFocus(id);
 }
 
-function Participant({ id, display, isPublisher, isLocalScreen, streamReady, focus, video }) {
+function renderVideo(id, ref, isPublisher, isLocalScreen, focus, dispatchFn) {
+  return (
+    <video
+      data-testid="participant-video"
+      autoPlay
+      ref={ref}
+      muted={isPublisher}
+      className={isPublisher && !isLocalScreen ? 'mirrored' : ''}
+      onClick={() => dispatchFn(toggleFocus(id, focus))}
+    />
+  );
+}
+
+function renderImage(id, focus, picture, dispatchFn) {
+  const placeholder = '/placeholder.png';
```

Comment:
I am not sure whether it will work if the application is deployed to a subdirectory.

Reply:
Right. Actually I added it first for testing and I forget to review / comment it.

Reply:
@imobachgs could you please fix this?